t/ui/18-tests-details.t

```diff
@@ -641,6 +641,23 @@ subtest 'additional investigation notes provided on new failed' => sub {
         'investigation status content shown as table');
 };
 
+subtest 'alert box shown if not already on first bad' => sub {
+    $driver->get('/tests/99940');
+    wait_for_ajax(msg => 'details tab for job 99940 loaded to test investigation');
+    $driver->find_element_by_link_text('Investigation')->click;
+    $driver->find_element("//div[\@class='alert alert-info']", 'xpath')
```

Comment:
Now is the only element with this class signature and it could be problematic if we add an other alert in the future. But now it's ok.

Reply:
don't care too much about the future, live in the present :)

http://www.extremeprogramming.org/rules/early.html